src/index.scss

```diff
@@ -3,12 +3,21 @@
 
 $nav-pills-border-radius: 0;
 $nav-pills-active-link-color: white;
-$nav-pills-active-link-bg: #373a3c;
+$nav-pills-active-link-bg: #366e98;
+$side-menu-bg: #044b7f;
+$side-menu-color: white;
```

Comment:
I think it would make sense to move these 2 vars somewhere else as they are not Bootstrap's.